app.js

```diff
@@ -36,45 +9,91 @@ function errorMessage(message) {
   }, 4000);
 }
 
-//  Add book
+class Book {
+  constructor(title, author, id) {
+    this.title = title;
+    this.author = author;
+    this.id = id;
+  }
 
-function addBook(title, author) {
-  const id = Date.now();
-  const bookList = {
-    id,
-    title,
-    author,
-  };
+  addBook() {
+    const { title, author, id } = this;
+    const bookList = { title, author, id };
+    books = JSON.parse(localStorage.getItem('books'));
+    if (title === '' || author === '') {
+      errorMessage('Kindly fill all inputs');
+    } else if (books !== null) {
+      books.push(bookList);
+      localStorage.setItem('books', JSON.stringify(books));
+      books = JSON.parse(localStorage.getItem('books'));
+    } else {
+      books = [];
+      books.push(bookList);
+      localStorage.setItem('books', JSON.stringify(books));
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+  }
+
+  remove() {
+    const { id } = this;
+    books = books.filter((book) => {
+      if (book.id === id) {
+        return false;
+      }
+      return true;
+    });
 
-  if (title === '' || author === '') {
-    errorMessage('Kindly fill all inputs');
-  } else {
-    books.push(bookList);
     localStorage.setItem('books', JSON.stringify(books));
-    displayBooks(bookList.id, bookList.title, bookList.author);
   }
 }
 
-// Global variable to check local storage
+// const book = new Book();
+function displayBooks(title, author, id) {
+  const bookList = document.querySelector('.book-list');
+  const items = document.createElement('li');
+  items.innerHTML += `
+  <div class="title-div"><h2><q>${title}</q></h2>
+  <h2> by </h2>
+  <h2> ${author}</h2></div>
+     `;
+  const removeBtn = document.createElement('button');
+  removeBtn.textContent = 'Remove';
+  removeBtn.className = 'removeBtn';
+  // removeBtn.id = id;
```

Comment:
- [ ] Kindly remove all commented line code, It is always a good practice to keep your code clean from such comments. 😉 💯

Reply:
I will check it out. Thank you.